src/drivers/wayland/lv_wayland_smm.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file lv_wayland_smm.h
+ *
+ */
+#ifndef LV_WAYLAND_SMM_H
```

Comment:
Please follow `lv_templ.h`'s format and add `#if LV_USE_WAYLAND` in the H files too.